src/wp-admin/plugin-install.php

```diff
@@ -79,18 +79,18 @@
 'id'		=> 'overview',
 'title'		=> __('Overview'),
 'content'	=>
-	'<p>' . sprintf( __('Plugins hook into ClassicPress to extend its functionality with custom features. Plugins are developed independently from the core ClassicPress application by thousands of developers all over the world. All plugins in the official <a href="%s">ClassicPress Plugin Directory</a> are compatible with the license ClassicPress uses.' ), __( 'https://wordpress.org/plugins/' ) ) . '</p>' .
+	'<p>' . sprintf( __('Plugins hook into ClassicPress to extend its functionality with custom features. Plugins are developed independently from the core ClassicPress application by thousands of developers all over the world. All plugins in the official <a href="%s">Plugin Directory</a> are compatible with the license ClassicPress uses.' ), __( 'https://wordpress.org/plugins/' ) ) . '</p>' .
```

Comment:
Here is something wrong. We say `ClassicPress Plugin Directory` but we link to `https://wordpress.org/plugins/`

Reply:
That is the old version. I changed that to say just Plugin Directory for now and it does go to the WP repo. It can be changed again when we get our directory working.

Reply:
But I think licence is OK since ours will be the same as WPs.

Reply:
>I think we could clarify more that this leads to WP Plugins and that those should by default actually NOT be used on CP installs or at least not preferably? Just a thought since we are editing this code part... we can as well add some clarification

(I think we crossed our comments, writing them at the same time)

Reply:
Changes made as per you suggestion in other comment.